app/src/main/java/fr/neamar/kiss/DataHandler.java

```diff
@@ -466,13 +466,13 @@ private Pojo getPojo(String id) {
 
     public void removeFromFavorites(Pojo pojo, Context context) {
         String favApps = PreferenceManager.getDefaultSharedPreferences(context).
-                getString("favorite-apps-list", "");
-        if (!favApps.contains(pojo.id + ";")) {
+                getString("favorite-apps-list", "" );
+        if (!favApps.contains(pojo.id + ";" )) {
             return;
         }
 
         PreferenceManager.getDefaultSharedPreferences(context).edit()
-                .putString("favorite-apps-list", favApps.replace(pojo.id + ";", "")).commit();
+                .putString("favorite-apps-list", favApps.replace(pojo.id + ";", "" )).commit();
```

Comment:
This file is only formatting changes, right? If so, please don't include it in your commit.